src/relay/qnn/util.h

```diff
@@ -67,6 +69,24 @@ static inline const int32_t GetQmax(const DataType& dtype) {
   }
 }
 
+Expr RequantizeLower(const Expr& input_tensor, const RequantizeAttrs* param,
+                     const Array<IndexExpr>& input_shape, const DataType& out_dtype);
+
+static inline Expr Requantize(const Expr& data, const Array<IndexExpr>& input_shape,
+                              const double& input_scale, const int32_t& input_zero_point,
+                              const double& output_scale, const int32_t output_zero_point,
+                              const DataType& out_dtype,
+                              const std::string& rounding = "TONEAREST") {
+  auto attrs = make_node<RequantizeAttrs>();
+  attrs->input_scale = std::move(input_scale);
+  attrs->input_zero_point = std::move(input_zero_point);
+  attrs->output_scale = std::move(output_scale);
+  attrs->output_zero_point = std::move(output_zero_point);
+  attrs->rounding = std::move(rounding);
+  attrs->out_dtype = std::move(out_dtype);
+  return RequantizeLower(data, attrs.operator->(), input_shape, out_dtype);
```

Comment:
does this mean we will create lowered ops directly, instead of `Requantize` op? and why?

Reply:
Yes, we directly lower to Relay ops. This utility will be used by ops during the FTVMLegalize for converting QNN ops to Relay ops. 

So, initially the qnn op will be `relay.qnn.op.concatenate` and after Legalize, the output will be the sequence of **only Relay ops**. Instead if we lower to `qnn.op.requantize` + `nn.concatenate`, we will again need to call Legalize to lower this requantize to Relay ops (which is against the purpose of Legalize).